locale/en/knowledge/advanced/buffers/how-to-use-buffers.md

```diff
@@ -33,22 +33,32 @@ In the wild, buffers are usually seen in the context of binary data coming from
 
 There are a few ways to create new buffers:
 
-    var buffer = new Buffer(8);
+```js
+var buffer = Buffer.alloc(8);
+// This will print out 8 bytes of zero:
+// <Buffer 00 00 00 00 00 00 00 00>
+```
+This buffer is uninitialized and contains 8 bytes of zero, notice that `new Buffer(size)` is deprecated since v6.0.0.
 
-This buffer is uninitialized and contains 8 bytes.
+Besides, we've also got [Buffer.allocUnsafe(size)](https://nodejs.org/dist/latest/docs/api/buffer.html#buffer_class_method_buffer_allocunsafe_size). This makes you get a buffer quickly without initializing it with zero values, so it's sometimes unsafe.
```

Comment:
As with the constructor, maybe don't mention it at all? People can find out about it in the docs if they really want to know, and that's where they should be going anyway. (I'm honestly not sure this doc is more valuable than the Buffer ref doc.)

Reply:
I think it should be a series of torturials, so it's easy to understand than professional, compared with Buffer ref doc.
But I'll remove it because of duplication.